tools/profiles/size.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Size doesn't really imply big or small. Maybe rename this file to ```small.json``` or something which implies that this is smaller than the default?